app/controllers/api/series_images_controller.rb

```diff
@@ -1,10 +1,17 @@
 # encoding: utf-8
 
 class Api::SeriesImagesController < Api::BaseController
-
   api_versions :v1
 
   filter_resources_by :series_id
 
   represent_with Api::ImageRepresenter
+
+  def resource
+    @series_image ||= series.try(:image) || super
```

Comment:
Did you confirm what announce will/won't do, when CRUD-ing these series/account/user images?

Now that I look, only StoryImages are announced at the moment.  Will need to add the others - maybe a separate issue/PR?